python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -368,6 +368,63 @@ async def seeing_loop(self):
                 )
                 break
 
+    async def do_gotoAltAz(self, data):
+        """Move to Alt/AZ position.
+
+        Parameters
+        ----------
+        data : A SALOBJ data object
+            Contains the data as defined in the SAL XML file.
+        """
+        self.assert_enabled()
+        self.log.debug(
+            f"NOT YET IMPLEMENTED : Moving DIMM to azimuth"
+        )
+        raise salobj.ExpectedError("Not implemented yet.")
+
+    async def do_gotoRaDec(self, data):
+        """Move to RA/DEC position.
+
+        Parameters
+        ----------
+        data : A SALOBJ data object
+            Contains the data as defined in the SAL XML file.
+        """
+        self.assert_enabled()
+        self.log.debug(
+            f"NOT YET IMPLEMENTED : Moving DIMM to target"
+        )
+        raise salobj.ExpectedError("Not implemented yet.")
+
+    async def do_changeDwellRate(self,data):
+        """Change how long to sample for in seconds.
+
+        Parameters
+        ----------
+        data : A SALOBJ data object
+            Contains the data as defined in the SAL XML file.
+        """
+        self.assert_enabled()
+        self.log.debug(
+            f"NOT YET IMPLEMENTED : Change dwell rate"
+        )
+        raise salobj.ExpectedError("Not implemented yet.")
+ 
+    async def do_changeMeasurementRate(self,data):
+        """Change sample integration.
+
+        Parameters
+        ----------
+        data : A SALOBJ data object
+            Contains the data as defined in the SAL XML file.
+        """
+        self.assert_enabled()
+        self.log.debug(
+            f"NOT YET IMPLEMENTED : Changing measurement rate"
```

Comment:
f-strings without placeholders are not allowed. Please remove the `f` at the start of the line.